app/models/story.rb

```diff
@@ -990,4 +993,10 @@ def fetched_attributes
 
     @fetched_attributes
   end
+
+  def check_for_description_repost(all_changes)
```

Comment:
Add an `attr_accessor` to `Story` (see line 141) to explicitly record + know here that this edit happened rather than infer it by checking for effects.

Reply:
done

Reply:
You can delete this method.

Reply:
Yes, sorry)